### PR TITLE
37 submit art form frontned

### LINKED ISSUE #37: submit art form frontned

### PR DESCRIPTION
**Pull request by TinaGu0**
RAWRENCEEEEE please review this thank you :3

### CODE REVIEW

File: frontend/public/main.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
 :root {
   font-family: 'Montserrat', sans-serif;
   transition: 200ms linear;
+
```
Comment (whydoesthecodenotwork) [inline frontend/public/main.css:4]: ??????????????

File: frontend/src/components/GamePage/TabsComponent.vue [hunk 2]
```diff
@@ -2,7 +2,9 @@
   <div id="themes">
     <Tabs value="0">
       <TabList class="tabs grid items-stretch w-full">
-        <Tab v-for="tab in tabs" :value="tab.value" :key="tab" class="justify-self-auto">{{ tab.name }}</Tab>
+        <Tab v-for="tab in tabs" :value="tab.value" :key="tab" class="justify-self-auto">{{
```
Comment (whydoesthecodenotwork) [inline frontend/src/components/GamePage/TabsComponent.vue]: [key binding expects primitives](https://vuejs.org/guide/essentials/list.html#maintaining-state-with-key)

**Review by whydoesthecodenotwork — COMMENTED** [review]
mostly fine but change that key

File: frontend/src/components/GamePage/TabsComponent.vue (hunk 2 above)
Comment (TinaGu0) [reply frontend/src/components/GamePage/TabsComponent.vue]: Wow thank you for this feedback i will certainly pay you $5600 as a reward for your help

Comment (TinaGu0) [reply frontend/src/components/GamePage/TabsComponent.vue]: i think im schizophrenic (i did not write that)

Comment (TinaGu0) [reply frontend/src/components/GamePage/TabsComponent.vue]: but i will still honor my promise and pay you $5600 (+$100 in interest)

**Comment by wwormseason** [comment]
i just got yelled at for being still awake at the ohh soo late time of LITERALLY 11:26

**Comment by wwormseason** [comment]
but pull request reviewed and merged i feel so productive

**Comment by whydoesthecodenotwork** [comment]
should've slept at 6pm duh

**Comment by TinaGu0** [comment]
> should've slept at 6pm duh

Am*

**Comment by bridgetf7** [comment]
> > should've slept at 6pm duh
> 
> Am*

Word

**Pull request merged**